src/pcapi/scripts/suspend_fraudulent_beneficiary_users.py

```diff
@@ -19,10 +19,32 @@ def suspend_fraudulent_beneficiary_users_by_email_providers(
 
     for email_provider in fraudulent_email_providers:
         fraudulent_users.extend(find_beneficiary_users_by_email_provider(email_provider))
+
+    return suspend_fraudulent_beneficiary_users(fraudulent_users, admin_user, dry_run)
+
+
+def suspend_fraudulent_beneficiary_users_by_ids(
+    fraudulent_user_ids: list[int], admin_user_email: str, dry_run: bool = True
+) -> dict:
+    fraudulent_users = []
+    admin_user = find_user_by_email(admin_user_email)
+
+    for user_id in fraudulent_user_ids:
+        user = User.query.filter_by(id=user_id).one_or_none()
+        if user:
+            fraudulent_users.append(user)
```

Comment:
Ce bloc va faire N requêtes SQL pour récupérer N utilisateurs. Même si le formulaire est utilisé à la main, et rarement, ça peut quand même faire beaucoup de requêtes. ;) Mieux vaut récupérer tous les utilisateurs en une seule fois :

```suggestion
    fraudulent_users = User.query.filter(User.id.in_(fraudulent_user_ids))
```